src/reducers/reducer.js

```diff
@@ -0,0 +1,15 @@
+import firstMessage from '../actions/actions';
+
+const initialState = {
+  message: 'Welcome',
+};
+const reducer = (state = initialState, action) => {
+  if (action.type === firstMessage.type) {
+    return {
+      message: `${action.value}`,
+    };
+  }
+  return initialState;
```

Comment:
Here you should even return the state you declared in parameters, to avoid a lot of eslint issues